perk-store-webapps/src/main/webapp/vue-app/components/perk-store-order/PerkstoreOrderApp.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <v-app
+    id="perkstoreOrderPortlet"
+    flat
+    dark>
+    <main>
+      <v-container pa-0>
+        <v-layout
+          row
+          wrap
+          mx-0
+          style="cursor: pointer"
+          @click="navigateTo('perkstore?notProcessedOrders=true')">
```

Comment:
use link instead of javascript even. In fact, this will increase UX by allowing user to open this in a new navigator tab (while with JS event, this will not be possible)

Reply:
Done!